packages/auth/src/openapi/id-provider.yaml

```diff
@@ -116,6 +116,8 @@ paths:
                 properties:
                   access:
                     $ref: ./auth-server.yaml#/components/schemas/access
+                  state:
+                    $ref: ./auth-server.yaml#/components/schemas/interact-state
```

Comment:
For some reason, my previous comment did not get posted.

Here is where you would move the definition of `state` instead of referencing it.

Reply:
Thank you for the review, I have made amendments based on your feedback.

Reply:
Thank you for your review, I have made amendments based on your feedback